inception-active-learning/src/main/java/de/tudarmstadt/ukp/inception/active/learning/sidebar/ActiveLearningSidebar.java

```diff
@@ -567,10 +569,12 @@ private boolean isAnnotatedInCas(LearningRecord aRecord)
                 .listAnnotationFeature(selectedLayer.getObject())) {
                 String annotatedValue = WebAnnoCasUtil
                     .getFeature(annotationFS, annotationFeature.getName());
-                if (annotatedValue.equals(aRecord.getAnnotation())) {
-                    highlightVID = new VID(WebAnnoCasUtil.getAddr(annotationFS));
-                    vMarkerType = ANNOTATION_MARKER;
-                    return true;
+                if (annotatedValue != null) {
+                    if (annotatedValue.equals(aRecord.getAnnotation())) {
```

Comment:
can be written in a single if:

```
if (annotatedValue != null && annotatedValue.equals(...))
```

or if `aRecord.getAnnotation()` is guaranteed to be non-null, then

```
if (aRecord.getAnnotation().equals(annotatedValue))
```